frontend/taipy/src/DataNodeViewer.tsx

```diff
@@ -1035,7 +1059,7 @@ const DataNodeViewer = (props: DataNodeViewerProps) => {
                                                                   format(dataValue as Date, "yyyy/MM/dd HH:mm:ss")
                                                                 : dtType == "float" && dtValue === null
                                                                 ? "NaN"
-                                                                : dtValue}
+                                                                : `${dtValue}`}
```

Comment:
Really??

Reply:
That's the consequence of the pseudo not ready ... Data not being recognized as tabular... Raised an error when trying to display! Wrong happen anymore